commcare_connect/opportunity/api/views.py

```diff
@@ -80,23 +82,23 @@ def post(self, *args, **kwargs):
 
         if OpportunityClaim.objects.filter(opportunity_access=opportunity_access).exists():
             return Response(status=200, data="Opportunity is already claimed")
-        if opportunity.remaining_budget < opportunity.budget_per_visit:
+        if opportunity.remaining_budget < opportunity.minimum_budget_per_visit:
```

Comment:
I think this needs to be max budget per visit (we dont want you claiming a job and not being able to get paid if you do the full delivery)

Reply:
The intention was to let user claim whatever the partial lowest budget visit if that was less than the remaining budget. But yeah, programmatically if want to allow only full delivery claiming, then I guess we should check that the remaining budget is more than the sum of budgets per all paymentunits (not just the Max).